src/valkeymodule.h

```diff
@@ -1058,7 +1076,12 @@ typedef ValkeyModuleScriptingEngineMemoryInfo (*ValkeyModuleScriptingEngineGetMe
     ValkeyModuleScriptingEngineSubsystemType type);
 
 /* Current ABI version for scripting engine modules. */
-#define VALKEYMODULE_SCRIPTING_ENGINE_ABI_VERSION 2UL
+/* Version Changelog:
+ *  - 1: Initial version.
+ *  - 2: Changed the `compile_code` callback to support binary data in the source code.
+ *  - 3: Added reset_env callback to reset both EVAL or FUNCTION scripts env.
```

Comment:
The `reset_env` callback isn't just added. The old reset_eval_env is replaced with this one.

Also the numbered list is formatted in a funny way, with bullet and number and colon. I think a number and a dot is a more common style.

```suggestion
 *  1. Initial version.
 *  2. Changed the `compile_code` callback to support binary data in the source code.
 *  3. Renamed reset_eval_env callback to reset_env and added an type parameter to be
       able to reset both EVAL or FUNCTION scripts env.
```